runtime/ops/os/sys_info.rs

```diff
@@ -352,14 +354,18 @@ pub fn os_uptime() -> u64 {
           .as_secs()
         })
         .unwrap_or_default();
+    } else {
+      panic!("result is not 0")
```

Comment:
Please revert these panic changes. It's just an uptime check, so if it fails we should just return `0`.